### ISSUE TITLE
Q: Command "python setup.py egg_info" failed with error code 1

### ISSUE THREAD

**Issue by LinuksGuru** [ReinerNippes/nextcloud_on_docker#72]
Hi,

I'm trying to install NextCLoud on Docker on fresh Debian 10.6
Prepare shell scripts throws this message at the end.

Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-e372lmw9/ansible/
I can trash this VM, no problem. May be I should install latest Ansible manually before running this shell script?

Thanks in advance.

------------------------------------------------------

```
Collecting ansible
  Using cached https://files.pythonhosted.org/packages/9c/f4/c156b10d7ae90ba6b99b1b126f7d30628adc1e733a6fbd63569852948f21/ansible-2.10.3.tar.gz
    Complete output from command python setup.py egg_info:
    
    
                ### ERROR ###
    
                Upgrading directly from ansible-2.9 or less to ansible-2.10 or greater with pip is
                known to cause problems.  Please uninstall the old version found at:
    
                /usr/lib/python3/dist-packages/ansible/__init__.py
    
                and install the new version:
    
                    pip uninstall ansible
                    pip install ansible
    
                If you have a broken installation, perhaps because ansible-base was installed before
                ansible was upgraded, try this to resolve it:
    
                    pip install --force-reinstall ansible ansible-base
    
                If ansible is installed in a different location than you will be installing it now
                (for example, if the old version is installed by a system package manager to
                /usr/lib/python3.8/site-packages/ansible but you are installing the new version into
                ~/.local/lib/python3.8/site-packages/ansible with `pip install --user ansible`)
                or you want to install anyways and cleanup any breakage afterwards, then you may set
                the ANSIBLE_SKIP_CONFLICT_CHECK environment variable to ignore this check:
    
                    ANSIBLE_SKIP_CONFLICT_CHECK=1 pip install --user ansible
    
                ### END ERROR ###
    
    
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-e372lmw9/ansible/

------------------------------------------------------

   Debian System ready to install Nextcloud.

ansible 2.7.7
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.7.3 (default, Jul 25 2020, 13:03:44) [GCC 8.3.0]

------------------------------------------------------

```

**Comment by ReinerNippes** [comment]
Hi, that should be fixed with commit d2b93016

**Comment by LinuksGuru** [comment]
OK thanks a lot !
